PatientRegistrationApp/src/components/SinglePatientHomeAddress.vue

```diff
@@ -57,7 +57,7 @@
 					required
 					:rules="[v => !!v || 'Zipcode field is required']"
```

Comment:
To apply the zip code validation rules, you need to change line 58 to:
:rules="postalCodeRules"

Otherwise, nothing happens :)